bin/rucio

```diff
@@ -386,15 +387,30 @@ def list_file_replicas(args):
     if args.missing:
         args.all_states = True
     client = get_client(args)
+
+    if args.selected_rse and args.rses:
+        # If the new cli arg and the old one are defined we don't know which one to take
+        logger.error("Cannot use --rse and --rses together, please use --rses")
+        return FAILURE
+    elif args.rse_expression and args.rses:
+        # If the new cli arg and the old one are defined we don't know which one to take
+        logger.error("Cannot use --expression and --rses together, please use --rses")
+        return FAILURE
+    elif args.selected_rse:
+        option_deprecation_message("--rse", "--rses")
+        client.get_rse(args.selected_rse)    # Raise an exception if RSE does not exist
+        args.rses = args.selected_rse
```

Comment:
I don't know if assigning to args can be problematic. Any reason for not using a local variable for the same purpose instead of assigning to args ?

Reply:
https://docs.python.org/3/library/argparse.html#argparse.Namespace

`args` is a `Namespace` object, it shouldn't be a problem.
`args` is our central store for options, so it felt natural to add it there and don't create an extra variable.

From my pov the developer could get confused when there is another local variable (like `rses`). Someone could use `args.rses` instead of the properly initialized variable. So the `args.rses` should at least be deleted after the local variable is initialized, so why not use it in the first place.

Reply:
Good for me :)